examples/pxScene2d/src/shell.js

```diff
@@ -155,12 +153,9 @@ if (false)
 
   scene.root.on("onPreKeyUp", function(e)
   {
-    console.log("in onPreKeyUp", e.keyCode, e.flags);
 	  var code  = e.keyCode;
```

Comment:
For the time being, let's leave the key prints in place as they are useful for debugging.